pkg/proto/nbcontract/v1/config.proto

```diff
@@ -56,10 +58,9 @@ message Configuration {
   bool is_sgx_node = 25;
   TeleportConfig teleport_config = 26;
   FeatureState ssh_status = 27;
-  FeatureState unattended_upgrade_status = 28;
+  optional FeatureState unattended_upgrade_status = 28;
```

Comment:
Curios about rationale / criteria for using optional with Feature State ...

Reply:
Feature state has an unspecified state. If it's not specified in the contract, it will be set as this state. In the VHD go binary, it will mostly handle this as "false".
I put this as optional because I think bootstrapper doesn't necessarily need to set this. If this is not set, it will be in unspecified state.
We will do a full review for all variables to see if they should be optional.
Does this sound make sense to you?

Reply:
Need to think some more about it, but wondering if in this case we need explicit "unspecified" value ...

Reply:
hmm. Per protobuff doc,
Do Include an Unspecified Value in an Enum

Reply:
Sounds good. Let's keep it this way and see if it feels right / we understand the tradeoffs.